tests/plugins/dominant-color-images/dominant-color-test.php

```diff
@@ -120,8 +138,8 @@ public function test_tag_add_adjust_to_image_attributes( $image_path, $expected_
 
 		foreach ( $expected_color as $color ) {
 			if ( false !== strpos( $color, $filtered_image_tags_added ) ) {
-				$this->assertStringContainsString( 'style="--dominant-color: #' . $expected_color . ';"', $filtered_image_tags_added );
-				$this->assertStringContainsString( 'data-dominant-color="' . $expected_color . '"', $filtered_image_tags_added );
+				$this->assertStringContainsString( 'style="--dominant-color: #' . $color . ';"', $filtered_image_tags_added );
+				$this->assertStringContainsString( 'data-dominant-color="' . $color . '"', $filtered_image_tags_added );
```

Comment:
This appears to be a bug fix, since `$expected_color` is an `array` and not the expected `string`. Nevertheless, tests weren't failing. So it seems the test is broken because the assertions weren't even being run.

Reply:
Ah, the problem is that the argument order for `strpos()` is incorrect. It should have been `strpos( $filtered_image_tags_added, $color )` instead. I've fixed this and switched to `str_contains()` in 4e8235f0.